s3.tf

```diff
@@ -9,4 +9,5 @@ resource "aws_s3_bucket" "devops_s3_course_2022_moved_resource" {
   versioning {
```

Comment:
you can add a variable for versioning, with default to false

Reply:
will do